specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -0,0 +1,5420 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/Tenant"
+            }
+          },
+          "403": {
+            "description":
+              "The tenant is not authorized to onboard to Azure Active Directory Connect Health because it does not meet the onboarding criteria."
+          }
+        }
+      },
+      "get": {
+        "tags": ["Configuration"],
+        "description": "Gets the details of a tenant onboarded to Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "configuration-GetTenant": {
+            "$ref": "./examples/Tenant.json"
+          }
+        },
+        "operationId": "configuration_GetTenant",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/Tenant"
+            }
+          },
+          "400": {
+            "description":
+              "Bad Request. This is the response code when the tenant is not onboarded to Azure Active Directory Connect Health.Exception will not be thrown."
+          }
+        }
+      },
+      "patch": {
+        "tags": ["Configuration"],
+        "description": "Updates tenant properties for tenants onboarded to Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "configuration-PatchTenant": {
+            "$ref": "./examples/PatchTenant.json"
+          }
+        },
+        "operationId": "configuration_PatchTenant",
+        "parameters": [
+          {
+            "name": "tenant",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Tenant"
+            },
+            "description": "The tenant object with the properties set to the updated value."
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/Tenant"
+            }
+          },
+          "404": {
+            "description":
+              "Not Found. This is the response code when the tenant is not onboarded to Azure Active Directory Connect Health.Exception will not be thrown."
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services": {
+      "get": {
+        "tags": ["Services"],
+        "description":
+          "Gets the details of services, for a tenant, that are onboarded to Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "service-GetServices": {
+            "$ref": "./examples/Services.json"
+          }
+        },
+        "operationId": "services_GetServices",
+        "parameters": [

```

Comment:
This is actually a list operation - so should probably be renamed accordingly and use the x-ms-pageable extension:
https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-pageable